utils/geometry.py

```diff
@@ -244,11 +244,18 @@ def get_flipped_line(self):
         return: Tline in flipped direction
         """
 
-        flipped_line = []
+        flipped_line = []      
+        
+        if hasattr(self, 'geoms'):            
+            for geom in self.geoms:
+                coords = []
+                for p in geom.coords:
+                    coords.insert(0, p)
+                flipped_line.insert(0, coords)                                                
+        else:
+            for p in self.coords:
+                flipped_line.insert(0, p)        
```

Comment:
dit kan, maar ik vermoed dat het volgende efficienter is:

if hasattr(self, 'geoms'):
    for geom in reversed(self.geoms):
        flipped_line.append([p for p in reversed(geom.coords)])
else:
    flipped_line.append([p for p in reversed(geom.coords)])